lib/semian/grpc.rb

```diff
@@ -0,0 +1,94 @@
+require 'semian/adapter'
+require 'grpc'
+
+module GRPC
+  GRPC::Unavailable.include(::Semian::AdapterError)
```

Comment:
It's possible we'd want to include it to some other errors, if they are indicative of a bad health of the remote service.

For example I see there is an `Internal` error class. If as I understand it it's similar to an HTTP 500, it might make sense to include it.

Not sure about the other ones, I know too little about the protocol to answer.